src/main/java/com/cloudbees/jenkins/plugins/bitbucket/JsonParser.java

```diff
@@ -62,10 +95,8 @@ public static String toJson(Object value) throws IOException {
 
     private static ObjectMapper createObjectMapper(){
         ObjectMapper mapper = new ObjectMapper();
-        SimpleDateFormat format = new SimpleDateFormat(DATE_FORMAT);
-        format.setTimeZone(TimeZone.getTimeZone("UTC"));
-        mapper.setDateFormat(format);
+        mapper.setDateFormat(new BitbucketDateFormat());
         mapper.configure(DeserializationConfig.Feature.FAIL_ON_UNKNOWN_PROPERTIES, false);
         return mapper;
     }
-}
+}
```

Comment:
Why did it deserve a final new line deletion 😇

Reply:
In my formatter I usually remove new line at the end. Is available an eclipse formatter for jenkins projects?